src/lib/php/Dao/ClearingDao.php

```diff
@@ -872,7 +872,7 @@ public function removeMainLicense($uploadId, $groupId, $licenseId)
    */
   function getIrrelevantFilesFolder(ItemTreeBounds $itemTreeBounds, $groupId, $onlyCurrent=true)
   {
-    $statementName = __METHOD__;
+    $statementName = __METHOD__.rand();
```

Comment:
This is a bad practice to add a random string to a unique statement name.

The function `getRelevantDecisionsCte()` alters the value for statement name based on the parameters, I don't think there are additional changes required.

Reply:
Actually this is not required, Somehow pushed. thanks for pointing.